canonical-websites/build.snapcraft.io
# Added a maintenance notification

## Done

Added a hardcoded maintenance notification to the main template.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)

## Screenshots

<img width="1078" alt="screen shot 2018-01-11 at 11 33 57" src="https://user-images.githubusercontent.com/83575/34821290-8223fdfe-f6c3-11e7-97d5-cbc0ef6e3c6c.png">
